subsys/nrf_security/src/drivers/cracen/cracenpsa/src/cipher.c

```diff
@@ -605,10 +605,10 @@ psa_status_t cracen_cipher_update(cracen_cipher_operation_t *operation, const ui
 					return silex_statuscodes_to_psa(sx_status);
 				}
 			} else {
-				psa_status_t r = initialize_cipher(operation);
+				psa_status_t result = initialize_cipher(operation);
```

Comment:
Nit but we do use something_status in most of the cases no? I would propose to update result -> status or psa_status.

Reply:
We already use `status` earlier in this function. I don't know why the decision was made to have two different `psa_status_t `return values here, but I figured that was a decision someone made so I didn't want to override it without digging into the code some more

Reply:
Hm, I don't really see why to be honest. I think it is safe to avoid redefining in this case and just use the existing status name.